src/MagnumPlugins/AssimpImporter/Test/AssimpImporterTest.cpp

```diff
@@ -625,6 +627,18 @@ void AssimpImporterTest::materialMultipleTextures() {
     }
 }
 
+void AssimpImporterTest::materialCoordinateSets() {
+    Containers::Pointer<AbstractImporter> importer = _manager.instantiate("AssimpImporter");
+    CORRADE_VERIFY(importer->openFile(Utility::Directory::join(ASSIMPIMPORTER_TEST_DIR, "material-coordinate-sets.dae")));
+
+    auto& mat = static_cast<PhongMaterialData&>(*importer->material(0));
+
+    // Assimp ignores ambient texture: CORRADE_COMPARE(mat.ambientCoordinateSet(), 1);
+    CORRADE_COMPARE(mat.diffuseCoordinateSet(), 2);
+    CORRADE_COMPARE(mat.ambientCoordinateSet(), 3);
+    CORRADE_COMPARE(mat.normalCoordinateSet(), 2);
+}
```

Comment:
Same as in case of the TinyGltf test -- ideally two variants, testing that the default behavior correctly fails. This also means you need to have ~~three~~ four different materials in order to verify each of the diffuse/specular/ambient/normal failures is done correctly. Instanced tests might (or might not) be helpful.

(What about a specular texture here?)

Reply:
I actually solved this before reading your comment... the second ambient should have been specular (copy pasta), and the failure is now also tested, but not separate. Maybe check really quick if the new version satisfies your requirements :)